src/sdam/topology.ts

```diff
@@ -986,7 +1073,7 @@ function isRetryableWritesSupported(topology: any) {
 }
 
 class ServerCapabilities {
-  constructor(ismaster: any) {
+  constructor(ismaster: Document) {
```

Comment:
I know it's a hard pill to swallow, but I think the same rationale applies here as it did with the "operation refactor" a week ago: we need a better solution for this, we shouldn't do it in this PR (I think likely we would want to add this to the `StreamDescription` class, since all operations are executed against `Connection` instances, also if we made these getters we'd have to explicitly save off another copy of the ismaster document. Let's solve this in the "Operation Layer Unification" project